resources/views/livewire/auth/login.blade.php

```diff
@@ -12,91 +12,92 @@
 use Livewire\Attributes\Validate;
 use Livewire\Volt\Component;
 
-new #[Layout('components.layouts.guest')] class extends Component {
-  #[Validate('required|string|email')]
-  public string $email = '';
+new #[Layout('components.layouts.guest')] class extends Component
+{
```

Comment:
why this change/reformat?

Reply:
I haven't explicitely do it myself. I don't know why my environnement does that :-(